client-lib/java/src/main/java/com/abcxyz/jvs/JVSClientBuilder.java

```diff
@@ -101,8 +103,18 @@ public JvsClient build() {
     updateConfigFromEnvironmentVars();
     configuration.validate();
 
+    URL url;
+    try {
+      // We have to convert endpoint to URL, as the JwkProviderBuilder
+      // appends its own path on the end if you simply pass the string.
+      url = new URL(configuration.getJvsEndpoint());
+    } catch (MalformedURLException e) {
+      throw new IllegalArgumentException(
+          String.format("endpoint was invalid %s", configuration.getJvsEndpoint()));
```

Comment:
Should we say something like "endpoint must be of the format http(s)://domain[:port]"?